fix(communities): add missing community image to import process

PascalPrecht opened the pull request:
This needs https://github.com/status-im/status-go/pull/2947

![Screenshot from 2022-11-04 15-04-15](https://user-images.githubusercontent.com/445106/199992794-a017f250-d663-4206-b3ea-229764b50610.png)

There's one issue here though:

We only get the image from the backend when the community has been created which takes a short moment.
So we should ideally show a loading indicator instead of the image.

@caybro if you have ideas how to make that work with our existing components, feel free to update the PR

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :interrobang: | 61590cbc | [#1](https://ci.status.im/job/status-desktop/job/branches/job/imports/job/PR-8132/1/) | 2022-11-04 14:13:49 | ~7 min | `e2e` | [:page_facing_up:`log`](https://ci.status.im/job/status-desktop/job/branches/job/imports/job/PR-8132/1/consoleText) |

caybro commented:
@PascalPrecht so we don't have this capability to show a loading animation together with an image, but we could show some placeholder image before we actually get the real one

PascalPrecht commented:
I"m fine with us doing that.
SmartIdenticon has a loader built-in for while the image is being requested. Thought we could leverage that somehow.

Let's go with whatever you think is feasable

PascalPrecht commented:
@caybro do you feel like taking this PR over then?

caybro commented:
> @caybro do you feel like taking this PR over then?

I'd say go with this current solution, then we can extend/improve it

caybro commented on ui/app/AppLayouts/CommunitiesPortal/popups/DiscordImportProgressContents.qml:
```diff
@@ -259,11 +259,12 @@ StatusScrollView {
-                source: Style.svg("contact") // TODO community icon
+                image.sourceSize: Qt.size(36, 36)
+                image.source: root.store.discordImportCommunityImage
+                color: "transparent"
```
This is default

PascalPrecht replied:
Updated.

PascalPrecht commented on ui/app/AppLayouts/CommunitiesPortal/popups/DiscordImportProgressContents.qml:
```diff
@@ -259,11 +259,18 @@ StatusScrollView {
+                Layout.preferredWidth: 24
+                Layout.alignment: Qt.AlignHCenter
+                visible: root.store.discordImportHasCommunityImage && root.store.discordImportCommunityImage.toString() == ""
+            }
```
@caybro added a loading indicator here in case we know there's gonna be an image but the progress signal hasn't emitted an image yet

caybro replied:
Perfect!

PascalPrecht commented:
@caybro might giving this one more review?
Updated according to your comment and added a `hasImage` property to control when to show a loading indicator.
We need this property because we can't rely on image == "". Reason being is that image == "" when there's no image in the first place **or** when the image hasn't been emitted from status-go yet.
